docker/docker-compose.yml

```diff
@@ -17,8 +17,29 @@ x-worker: &base-worker
 version: "2.4"
 services:
 
+  traefik:
+    image: traefik:v2.4
+    container_name: traefik
+    command: >
+      --api.insecure=true
```

Comment:
totally new to traefik so maybe need a clarification here on the purpose of API besides debugging and the dashboard.  Is it needed for load balancing.
https://doc.traefik.io/traefik/operations/api/

Reply:
No.  We can disable it.